pomgen.py

```diff
@@ -81,11 +82,13 @@ def main(args):
     if args.verbose:
         logger.debug("Global pom content: %s" % pom_content)
 
+    override_file_info = overridefileinfo.OverrideFileInfo(cfg.override_file_paths, repo_root)
     mvn_install_info = maveninstallinfo.MavenInstallInfo(cfg.maven_install_paths)
     ws = workspace.Workspace(repo_root, cfg,
```

Comment:
Note that this is not the only place workspace is instantiated.  Also see query.py and misc/extedeps_pomgen.py